src/utils/peopleHelper.ts

```diff
@@ -0,0 +1,71 @@
+import { Person } from '../types';
+import { Filter } from './Filter';
+
+export const findChild = (people: Person[], parentName: string) => {
+  const findPerson = people.find(person => person.name === parentName);
+
+  return findPerson;
+};
+
+export const columsByTable = [
+  'Name',
+  'Sex',
+  'Born',
+  'Died',
+  'Mother',
+  'Father',
+];
+
+export const genderButtons = ['Male', 'Female'];
+
+export const centuryButtons = ['16', '17', '18', '19', '20'];
+
+export const getPreparedPeople = (people: Person[], fields: Filter) => {
+  const { sex, query, centuries, sort, order } = fields;
+  let peopleList = [...people];
+
+  if (sex) {
+    peopleList = peopleList.filter(person => person.sex === sex);
+  }
+
+  const normalizeQuery = query.toLowerCase().trim();
+
+  if (normalizeQuery) {
+    peopleList = peopleList.filter(person => {
+      return (
+        person.name.toLowerCase().includes(normalizeQuery) ||
```

Comment:
`.toLowerCase().includes(normalizeQuery)` we can move it to separated function and reuse